### ISSUE TITLE
Missing raw_counts_genomes.tsv

### ISSUE THREAD

**Issue by philippbayer** [metagenome-atlas/atlas_analyze#10]
Hi!

I was just trying to play around with the scripts here, is this repo still in use?

I'm now getting this error:
```
MissingInputException in rule get_mapping_rate  in line 92 of /scratch/pawsey0390/pbayer/seagrass_atlas/atlas_analyze/Snakefile:
Missing input files for rule get_mapping_rate:
    output: Results/mapping_rate.tsv
    affected files:
        genomes/counts/raw_counts_genomes.tsv
```

As it seems like the rules in binning.smk dropped tsv files in favor of parquet files. I guess that wasn't updated here?

**Comment by SilasK** [comment]
No, the repo is no longer in use. A somewhat updated scripts R and pyhton are in https://github.com/metagenome-atlas/Tutorial

However they are not fully up to date. e.g. I now use parquet files instead of tsvs

I hope to update the tutorial soon, but probably not befopre the end of the year..
